README.md

```diff
@@ -45,11 +45,9 @@ file_path_or_url = 'path/to/file.mp4'
 # process of fixing this though.
 player = OMXPlayer(file_path_or_url)
 
-# The player will initially be paused
-
-player.play()
-sleep(5)
 player.pause()
```

Comment:
The player don't pause initially and it's not my changes. I just fixed the readme.